src/git-source-provider.ts

```diff
@@ -19,6 +19,17 @@ export async function getSource(settings: IGitSourceSettings): Promise<void> {
   )
   const repositoryUrl = urlHelper.getFetchUrl(settings)
 
+  // Determine the default branch
+  if (!settings.ref && !settings.commit) {
+    core.startGroup('Determine the default branch')
```

Comment:
Super nit:

All of our other groups names using the present progressive verb tense `Initializing the repository' `Initializing the repository'` `Getting Git version info`

Can we update this to ` Determining the default branch`